extension/src/paymentHandler/payment-handler.js

```diff
@@ -158,65 +146,31 @@ function _requiresLineItems(makePaymentRequestObj) {
     return addCommercetoolsLineItemsFlag
   }
 
-  if (_isKlarna(makePaymentRequestObj) || _isAffirm(makePaymentRequestObj))
```

Comment:
Don't we need Klarna and Affirm payments? If we need them, then we need lineItems for these payments.

Reply:
Now we don't need to know to determine whether line items are required. Indeed the payment method is not yet chosen by shopper at the moment when the payment session is created.

Here are reply from Adyen support

> Unlike /payments endpoint, you only submit the payments call after you get the payment method selection from Shopper. For /sessions endpoint, you will need to send all the required filed when you submit /sessions request, for example, lineItems are compulsory filed for certain payment methods such as Klarna, afterpay, etc. In order for these payment methods to be shown to shopper in the checkout page, you will need to send lineItems in the /sessions call. Whether shopper chooses to pay with those payment methods or not, that will be handled within our component itself.